lms/db/__init__.py

```diff
@@ -59,13 +59,10 @@ def update_from_dict(self, data, skip_keys=None):
                 setattr(self, key, data[key])
 
     def __repr__(self):
-        return "{class_}({kwargs})".format(  # pylint: disable=consider-using-f-string
-            class_=self.__class__.__name__,
-            kwargs=", ".join(
-                f"{kwarg}={repr(getattr(self, kwarg))}"
-                for kwarg in self.__table__.columns.keys()  # pylint:disable=no-member
-            ),
+        kwargs = ", ".join(
+            f"{column}={repr(getattr(self, column))}" for column in self.columns()
```

Comment:
To be honest I've never seen this top level `__repr__` before. This works better and it's sorter.